frontend/public/components/create-yaml.tsx

```diff
@@ -7,18 +7,17 @@ import { connectToPlural } from '../kinds';
 import { AsyncComponent } from './utils/async';
 import { Firehose, LoadingBox } from './utils';
 import { K8sKind, apiVersionForModel, referenceForModel } from '../module/k8s';
-import { ErrorPage404 } from './error';
 import { ClusterServiceVersionModel } from '../models';
 
 export const CreateYAML = connectToPlural((props: CreateYAMLProps) => {
-  const {match, kindsInFlight, kindObj} = props;
+  const {match, kindsInFlight, kindObj, download = true, showHeader = true} = props;
   const {params} = match;
 
   if (!kindObj) {
     if (kindsInFlight) {
       return <LoadingBox />;
     }
-    return <ErrorPage404 />;
+    return <AsyncComponent loader={() => import('./edit-yaml').then(c => c.EditYAML)} create={true} showHeader={showHeader} download={download} />;
```

Comment:
I think it's better to just create a separate ImportYAML component since almost all the logic is in the EditYAML component anyway.